source/crud/tests/v2/default-write-concern.yml

```diff
@@ -0,0 +1,413 @@
+data:
+  - _id: 1
+    x: 11
+  - _id: 2
+    x: 22
+collection_name: default_write_concern_coll
```

Comment:
Consider the formatting used in other `v2` tests, for conciseness. Small documents can be expressed on one line (e.g. `- { _id: 1, x: 11 }`). Likewise for arguments and pipelines in actual tests.

Also, the collection and database names can be turned into references to reduce duplication of the original string. For instance, `aggregate-merge.yml` uses `collection_name: &collection_name 'test_aggregate_merge'`.